### ISSUE TITLE
"TypeError: NetworkError when attempting to fetch resource" => Source Map URL: browser-polyfill.js.map

### ISSUE THREAD

**Issue by jnachtigall** [dequelabs/axe-firefox-devtools#57]
This really otherwise *really* nice add-on pollutes my Console ;) with a NetworkError message:
![image](https://user-images.githubusercontent.com/3307404/33262219-ddb4294a-d365-11e7-8ed3-d6d4dd3f27a4.png)

Also note that the `axe` add-on is absent in these cases from the DevTools tab panel. So I guess it can also not be initialized in these cases.

You can see the error at https://jnachtigall.github.io/bp/bp.html (a otherwise totally unrelated project).

FWIW, I used Firefox Nightly 59.0a1 (2017-11-26) (64-bit) on Windows 10, but this should matter here.

**Comment by mikoMK** [comment]
I can confirm this error. However in my case the "aXe" tab is still visible, accessible and operational.
Firefox Developer Edition 59.0b11 (64-Bit) on Windows 10

**Comment by jnachtigall** [comment]
Yes, "aXe" still works, but the warning is very annoying when using the DevTools. So annoying that I always disable aXe and always reinstall on point when I need it ;)

**Comment by mikoMK** [comment]
Ah sorry. So I misunderstood your initial message. I thought your panel is absent **because of** the issue :-)

**Comment by mtelgkamp** [comment]
Please remove the last line in browser-polyfill.js to prevent this warning.

[axe-firefox-devtools_issues_57.patch.txt](https://github.com/dequelabs/axe-firefox-devtools/files/1774142/axe-firefox-devtools_issues_57.patch.txt)

**Comment by jnachtigall** [comment]
@mtelgkamp Or alternatively, add this source map file to the repo. You think you could send in a PR?

**Comment by dylanb** [comment]
@marcysutton @wilco seems like a small change that we could make to fix this?

**Comment by marcysutton** [comment]
We don't want to modify the polyfill as it gets downloaded from Mozilla. Instead, we've added a copy task to our build system to include the sourcemap. It will be fixed internally and published in a future update.
